core/src/main/java/com/datastax/oss/driver/api/core/cql/StatementBuilder.java

```diff
@@ -86,24 +91,28 @@ public T withRoutingKeyspace(CqlIdentifier routingKeyspace) {
    * Shortcut for {@link #withRoutingKeyspace(CqlIdentifier)
    * withRoutingKeyspace(CqlIdentifier.fromCql(routingKeyspaceName))}.
    */
-  public T withRoutingKeyspace(String routingKeyspaceName) {
+  @NonNull
+  public T withRoutingKeyspace(@NonNull String routingKeyspaceName) {
```

Comment:
Most arguments are nullable if you initialized the builder by copying another statement, and need to clear some of the attributes.